src/App.js

```diff
@@ -2,14 +2,29 @@ import React from "react";
 
 import Bookings from "./Bookings";
 import "./App.css";
+import Heading from "./Heading";
+import TouristInfoCard from "./TouristInfoCards";
+import Footer from "./Footer";
+import Restaurant from "./Restaurant";
+
 
 const App = () => {
+  const contactDetails = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789",
+  ];
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCard />
       <Bookings />
+      <Restaurant />
+      <Footer props={contactDetails} />
```

Comment:
Its best not to call a prop, `props`. What you could do here is `{...contactDetails}` this would pass all of the contact details down to the Footer as seperate props. Otherwise in your component you have `props.props.xxx` which loooks a bit rubbish! 😅